### ISSUE TITLE
Make non-square album art display better on "Albums"

### ISSUE THREAD

**Issue by Crono23** [ubuntu-flutter-community/musicpod#200]
Hi!
I have some music file with a non-square rectangle shaped album art. It doesn't look great on the "Albums" section, but is fine everywhere else. It would be great if this is fixed. Thanks! :)

[Albums section](https://www.dropbox.com/scl/fi/cum8yfhnhhk95thc6vve3/Screenshot-from-2023-10-16-03-36-31.png?rlkey=25mp8ret19fkfcfdj3rhtlcqv&dl=0)
[Artists section](https://www.dropbox.com/scl/fi/kij1gtsttlr849xsx2js4/Screenshot-from-2023-10-16-03-37-09.png?rlkey=ercynj55efxb6rexr44xxtyjd&dl=0)
[Inside the album](https://www.dropbox.com/scl/fi/00r3xqs3ut0xvnabe64o9/Screenshot-from-2023-10-16-03-37-17.png?rlkey=bc8dxclzy3kyqjv2ualoye3tm&dl=0)
[Maximized view](https://www.dropbox.com/scl/fi/egatqamufsjff71y6dsim/Screenshot-from-2023-10-16-03-51-11.png?rlkey=be7pmzx2h16ouqxmll2zad8jo&dl=0)

**Comment by Feichtmeier** [comment]
Hi! Thanks for using musicpod! What language are you using? Eventually could you add translations?

on topic: what would you prefer as an improvement? stretch the art to fit into the rounded rectangle or display the name underneath instead of on top (stacked) ?

**Comment by Crono23** [comment]
> Hi! Thanks for using musicpod! What language are you using? Eventually could you add translations?

I mainly use English, but I could help on Indonesian translations if needed.

> on topic: what would you prefer as an improvement? stretch the art to fit into the rounded rectangle or display the name underneath instead of on top (stacked) ?

I was mainly concerned with the first, but the latter also sounds good! Both of them would be nice to have.

**Comment by Feichtmeier** [comment]
@Crono23 

![grafik](https://github.com/ubuntu-flutter-community/musicpod/assets/15329494/a658ca30-e539-4539-a59a-10d530afbc12)

?

**Comment by Crono23** [comment]
> @Crono23
> 
> ![grafik](https://user-images.githubusercontent.com/15329494/280086518-b6ae0b2b-31ea-4a56-90c4-672ee72ef3fc.png)
> 
> ?

Ooo that looks nice!

**Comment by Feichtmeier** [comment]
Cool :) Have a look after you refreshed the snap if it works now. I also changed how the artwork is aligned inside those cards. Often it works but ofc it does not work for every single artwork size

**Comment by Crono23** [comment]
Just had the chance to check it out now and it is lovely. Thanks for the improvement!

As for the artwork. I did a mock-up in Krita finding ways to make the vertical non-square art prettier. I added images of a disc behind them so it's like a disc case.
![Album edit](https://github.com/ubuntu-flutter-community/musicpod/assets/88884864/8604978a-02f6-4680-a8f1-32b6610b07d7)
Another version with highlight and shadow:
![Album edit 2](https://github.com/ubuntu-flutter-community/musicpod/assets/88884864/06322684-87a6-46d9-8b63-2d85623344b0)
I'm thinking if there's a way to display these disc image behind them when the artworks aren't square.

However, I am not sure if that is an improvement over simply zooming the art as in the case with Lollypop here:
![Lollypop](https://github.com/ubuntu-flutter-community/musicpod/assets/88884864/d9313f51-a5f2-4238-b6fa-4b677302897c)
Although, zooming it can obscure the art a bit like how it hides Faye Wong's eyes in "Eyes on Me" art lol.

I am no programmer so I don't know if this complicates things. Let me know what you think!
